test/__snapshots__/index.test.js.snap

```diff
@@ -1856,30 +1855,30 @@ esnext-ts
 
 
 Build \\"esnext-ts\\" to dist:
-1010 B: esnext-ts.js.gz
-899 B: esnext-ts.js.br
-1011 B: esnext-ts.esm.mjs.gz
-896 B: esnext-ts.esm.mjs.br
-1072 B: esnext-ts.umd.js.gz
-959 B: esnext-ts.umd.js.br"
+1180 B: esnext-ts.js.gz
+1055 B: esnext-ts.js.br
+1181 B: esnext-ts.esm.mjs.gz
+1056 B: esnext-ts.esm.mjs.br
+1240 B: esnext-ts.umd.js.gz
+1112 B: esnext-ts.umd.js.br"
```

Comment:
TODO: Sizable regression here, need to figure out what this is

Reply:
Ah, that's [`AsyncFromSyncIterator`](https://github.com/zloirock/core-js/blob/master/packages/core-js/internals/async-from-sync-iterator.js) -- going to call this fair enough.